src/sonic-yang-models/sonic-vlan.yang

```diff
@@ -0,0 +1,186 @@
+module sonic-vlan {
+    namespace "http://github.com/Azure/sonic-vlan";
+    prefix svlan;
+    yang-version 1.1;
+
+    import sonic-common {
+        prefix scommon;
+    }
+
+    import sonic-port {
+        prefix prt;
+    }
+
+    import sonic-portchannel {
+        prefix spc;
+    }
+
+    organization
+        "SONiC";
+
+    contact
+        "SONiC";
+
+    description
+        "SONIC VLAN";
+
+    revision 2019-05-15 {
+        description
+            "Initial revision.";
+    }
+
+
+    container sonic-vlan {
+
+        container VLAN { 
+
+            list VLAN_LIST { 
+                key "name";
+                must "./name = concat('Vlan', string(./vlanid))"{
+                    error-app-tag vlan-invalid;
+                }
+
+                leaf name {
+                    type string {
+                        pattern "Vlan(409[0-5]|40[0-8][0-9]|[1-3][0-9]{3}|[1-9][0-9]{2}|[1-9][0-9]|[1-9])" {
+                            error-message "Invalid Vlan name pattern";
+                            error-app-tag vlan-name-invalid;
+                        }
+                    }
+                }
+
+                leaf vlanid {
+                    mandatory true;
+                    type uint16 {
+                        range "1..4095" {
+                            error-message "Vlan ID out of range";
+                            error-app-tag vlanid-invalid;
+                        }
+                    }
+                }
+
+                leaf-list members {
```

Comment:
Why do we need this field, when there is a separate table for VLAN_MEMBERS?

Reply:
I think, this is for backward compatibility. 
With change in SONIC Config, The challange is to ask NetOPS to change few tools which are responsible to deploy config on thousands of switches. So if we were using it in data Center previously then usually we try to be backward compatible.